cli/lib/context/context_open_in_browser.dart

```diff
@@ -0,0 +1,16 @@
+import 'dart:io';
+
+import 'package:stax/context/context.dart';
+
+extension ContextOpenInBrowser on Context {
+  void openInBrowser(String url) {
+    final openCommand = Platform.isWindows
+        ? ['PowerShell', '-Command', '''& {Start-Process "$url"}''']
+        : ['open', url];
+
+    command(openCommand)
```

Comment:
Here let's just have `return` without follow up `announce`, `runSync`, or `printNotEmptyResultFields` 
```suggestion
    return command(openCommand);
```